src/app/embed/adapters/feed.adapter.ts

```diff
@@ -0,0 +1,88 @@
+import { EMBED_FEED_URL_PARAM, EMBED_FEED_ID_PARAM, EMBED_EPISODE_GUID_PARAM } from './../embed.constants'
+import {Injectable} from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import { AdapterProperties, DataAdapter } from './adapter.properties';
+import { sha1 }  from './sha1'
+
+const GUID_PREFIX = 's1!'
+@Injectable()
+export class FeedAdapter implements DataAdapter {
+	private feedUrl: string
+	private feedId: string
```

Comment:
Is this `feedId` specific to Draper?  Or would it ever have uses outside of that?